onnx/onnxifi.h

```diff
@@ -38,13 +38,13 @@ extern "C" {
 #endif
```

Comment:
please don't mix too much formatting in one PR.

Reply:
Will do the change outside VS which often applies formatting automatically

---
In reply to: [344910141](https://github.com/onnx/onnx/pull/2437#discussion_r344910141) [](ancestors = 344910141)